pyproject.toml

```diff
@@ -0,0 +1,17 @@
+[build-system]
+requires = ["hatchling"]
```

Comment:
what's hatchling? is that what we use in python projects?

Reply:
it's the build system used in the 'build your first python project' tutorial I followed

very open to changing this to anything else

Reply:
i don't think you need a build system for any reason. would prefer to package this more similarly to our other python projects at sentry, rather than bring in unfamiliar dependencies.